source/dubregistry/registry.d

```diff
@@ -248,6 +250,44 @@ class DubRegistry {
 		return ret;
 	}
 
+	PackageInfo[string] getPackageInfoWithDependencies(string packname, bool include_errors, bool minimize)
+	{
+		PackageInfo[string] ret;
+		getPackageInfoWithDependencies(packname, include_errors, minimize, ret);
+		logDebug("%s", ret.keys);
```

Comment:
Maybe add a tag to grep for it in the log?
e.g. `getPackageInfoWithDependencies[keys]: `